File: java/oop/InterfaceApp/src/com/techlabs/icrudable/VendorDB.java
package com.techlabs.icrudable;

public class VendorDB implements ICRUDable{
	
	public void create(){
		System.out.println("VendorDB is created");
	}
	public void update(){
		System.out.println("VendorDB is updated");
	}
	public void read(){
		System.out.println("VendorDB is Showing");
	}
	public void delete(){
		System.out.println("VendorDB is deleting");
	}
}

File: dotNet/CSharp/InterFaceCRUDApp/InterFaceCRUDApp/VendorDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterFaceCRUDApp
{
    public class VendorDB : ICRUDAble
    {
        public void Create()
        {
            Console.WriteLine("VendorDB is Created");
        }

        public void Delete()
        {
            Console.WriteLine("VendorDB is Deleting");
        }

        public void Read()
        {
            Console.WriteLine("VendorDB is Reading");
        }

        public void Update()
        {
            Console.WriteLine("VendorDB is Updating");
        }
    }
}
